src/classes/issues-processor.ts

```diff
@@ -536,8 +541,10 @@ export class IssuesProcessor {
     issue: Issue,
     staleLabel: string,
     actor: string,
+    addLabelsWhenUpdatedFromStale: Readonly<string>[],
+    removeLabelsWhenUpdatedFromStale: Readonly<string>[],
     closeMessage?: string,
-    closeLabel?: string
+    closeLabel?: string,
```

Comment:
Nit: remove comma

Reply:
If you run `npm run all` and no linter is altering this comma, keep it like this it's fine ;)